tests/measurements/legacy/test_classical_shadow_legacy.py

```diff
@@ -120,20 +119,6 @@ class TestClassicalShadow:
     shots_list = [1, 100]
     seed_recipes_list = [None, 74]  # random seed
 
-    @pytest.mark.parametrize("shots", shots_list)
```

Comment:
No need to have special legacy device tests anymore, since doesn't depend on the device.